.travis.yml

```diff
@@ -47,7 +47,7 @@ matrix:
         - export PATH="$HOME/miniconda/bin:$PATH"
         - cd doc
         - pip install -q -r requirements-doc.txt
-        - pip install -q yapf==0.23.0 sphinx-gallery
+        - pip install -q yapf==0.23.0
```

Comment:
This installs `sphinx-gallery` in Travis, but readthedocs doesn't have access to it, so readthedocs fails.